runtime/lua/vim/lsp/util.lua

```diff
@@ -267,7 +426,6 @@ function M.apply_text_document_edit(text_document_edit, index)
   if should_check_version and (text_document.version
       and M.buf_versions[bufnr]
       and M.buf_versions[bufnr] > text_document.version) then
-    print("Buffer ", text_document.uri, " newer than edits.")
```

Comment:
Why are we removing this? It seems like we will silently fail now to apply text edits, when we probably want to let users know.